doc/.custom_wordlist.txt

```diff
@@ -10,6 +10,7 @@ AppArmor
 ARMv
 ARP
 ASN
+Auth
```

Comment:
Should this be `Auth0` and not `Auth`?

Reply:
The spellchecker was complaining about `Auth` appearing in many pages. I suspect this is the navigation item and the number stripped by the checker.